### PR TITLE
CI: add a flatpak job

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
The job is running on my fork at https://github.com/bilelmoussaoui/pithos/runs/1339100384

### CODE REVIEW

File: flatpak/io.github.Pithos.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "app-id": "io.github.Pithos",
   "runtime": "org.gnome.Platform",
-  "runtime-version": "master",
+  "runtime-version": "3.38",
```
Comment (BrainBlasted) [inline flatpak/io.github.Pithos.json:4]: Why 3.38?

Comment (bilelmoussaoui) [reply flatpak/io.github.Pithos.json:4]: Why not? the GNOME docker image could be used though which should work fine I guess. But as Pithos is not really targeting the latest GNOME, it makes sense to use the latest stable version.

Comment (TingPing) [reply flatpak/io.github.Pithos.json:4]: I agree we don't really target master, this seems reasonable.

**Comment by TingPing** [comment]
Just throwing a thought out there:

I would love to remove the LGTM and SideCI integrations for a generic GitHub action that runs a python linter.

**Pull request merged**